File: patterns/c-patterns/pattern6.c
// Pattern 6: Triangle

//     *
//    * *
//   * * *
//  * * * *

#include <stdio.h>
#include <conio.h>
int main()
{
    int i, j, k;

    int p_height = 5;

    for (i = 1; i < p_height; i++)
    {
        for (k = p_height - 1; k >= i; k--)
        {
            printf(" ");
        }
        for (j = 1; j <= i; j++)
        {
            printf("* ");
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern6.js
// Pattern 6

//     *
//    * *
//   * * *
//  * * * *

const p_height = 5;

for (let i = 1; i < p_height; i++) {
    let spaces = '';
    for (let k = p_height - 1; k >= i; k--) {
        spaces += ' ';
    }

    let stars = '';
    for (let j = 1; j <= i; j++) {
        stars += '* ';
    }

    console.log(spaces + stars);
}